twitcher/owsproxy.py

```diff
@@ -154,7 +199,8 @@ def owsproxy_delegate(request):
     Delegates owsproxy request to external twitcher service.
     """
     twitcher_url = request.registry.settings.get('twitcher.url')
-    url = twitcher_url + '/ows/proxy'
+    protected_path = request.registry.settings.get('twitcher.ows_proxy_protected_path', '/ows')
+    url = twitcher_url + protected_path + '/proxy'
```

Comment:
Will planned (in some future...?) `/twitcher/ows/proxy/<name>` => `/<name>` be affected by this hardcoded value?

Reply:
Yes I'm a little bit worried about that too. This is done like that on the birdhouse side... We will probably be able to redirected from nginx / to /twitcher/ows/proxy though.